src/app/components/FeaturedCourses/featured-courses.tsx

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import "./featured-courses.css";
+import Image from 'next/image'
+import courseone from '/public/courseone.svg'
+import coursetwo from '/public/coursetwo.svg'
+import coursethree from '/public/coursethree.svg'
+import coursefour from '/public/coursefour.svg'
+import kevin from '/public/kevin.svg'
+import star from '/public/star.svg'
+import user from '/public/user.svg'
+import barchat from '/public/barchat.svg'
+import clock from '/public/clock.svg'
+
+function FeaturedCourses() {
+    return (
+        <div className='featured-section'>
+            <div className='our-featured-box'>
+                <h2>Our featured courses</h2>
+                <p>Vestibulum sed dolor sed diam mollis maximus vel nec dolor. Donec varius purus et eleifend porta.</p>
+            </div>
+            <div className='featured-section-box'>
```

Comment:
FeatureCourseCard adında bir component oluşturulup card için gerekli veriler props olarak dinamik alınacak. Bu component verilen data kadar card gösterecek şekilde dinamik çalışmalı.
- Component klasörü src içerisinde olacak app içerisinden çıkartılacak.